deployment/docker-compose/README.md

```diff
@@ -0,0 +1,458 @@
+    - [Installation With Black Duck](#installation-with-black-duck)
+- [Upgrading Alert](#upgrading-alert)
+    - [Standalone Upgrade](#standalone-upgrade)
+    - [Upgrade With Black Duck](#upgrade-with-black-duck)
+- [Certificates](#certificates)
+- [Environment Variables](#environment-variables) 
+    - [Edit Environment File](#edit-environment-file)
+    - [Edit the Overrides File](#edit-the-overrides-file)
+    - [Environment Variable Overrides](#environment-variable-overrides)
+    - [Alert Hostname Variable](#alert-hostname-variable)
+    - [Alert Logging Level Variable](#alert-logging-level-variable)
+    - [Email Channel Environment Variables](#email-channel-environment-variables)
+    - [Environment Variable Classifications](#environment-variable-classifications)
+- [Advanced Configuration](#advanced-configuration)
+    - [Changing Server Port](#changing-server-port)
+    - [Changing Memory Settings](#changing-memory-settings)
+
+## Requirements
+
+- You have a Docker host with at least 2GB of allocatable memory.
+- You have administrative access to your docker host.  
+
+## Installing Alert
+Deployment files for Docker Compose are located in the docker-compose directory of the: 
+
+blackduck-alert-\<VERSION>\-deployment.zip file.
+
+- Extract the contents of the ZIP file.
+- For installing with the Black Duck the files are located in the hub sub-directory.
+- For installing Alert standalone the files are located in the standalone sub-directory.
+
+### Standalone Installation
+This section walk through the instructions to install Alert in a standalone fashion.
+#### Overview
+1. Create a directory for secrets.
+2. Create ALERT_ENCRYPTION_PASSWORD file.
+3. Create ALERT_ENCRYPTION_GLOBAL_SALT file.
+4. Manage certificates.
+5. Modify environment variables.
+6. Bring the containers up.
+ 
+#### Details 
+This section walks through each step of the installation procedure.
+
+##### 1. Create a directory for secrets.
+
+- Create a directory to store secrets.
+    ```
+    mkdir -p <PATH>
+    ```
+    
+    Example: 
+    ```
+    mkdir -p /alert/mysecrets
+    ```
+    
+- Uncomment the following from the docker-compose.local-overrides.yml file alert service section.
+    ```
+      volumes: ['<PATH_TO_SECRETS>:/run/secrets']
+    ```
+      
+- Replace <PATH_TO_SECRETS> of the docker-compose.local-overrides.yml file with the directory just created.
+    
+    Example:
+    ```
+        volumes:['/alert/mysecrets:/run/secrets']
+    ```
+##### 2. Create ALERT_ENCRYPTION_PASSWORD file.
+  
+- Create a file containing the encryption password for Alert in the secrets directory.
+
+    ```
+    echo "<PASSSWORD_TEXT>" >> <PATH_TO_SECRETS>/ALERT_ENCRYPTION_PASSWORD
+    ```
+
+##### 3. Create ALERT_ENCRYPTION_GLOBAL_SALT file.
+
+- Create a file containing the encryption salt for Alert in the secrets directory.
+
+    ```
+    echo "<SALT_TEXT>" >> <PATH_TO_SECRETS>/ALERT_ENCRYPTION_GLOBAL_SALT
```

Comment:
We should mention they need to replace <SALT_TEXT> with the value they want to use